lib/skate_web/components/layouts/barebones.html.heex

```diff
@@ -29,14 +29,33 @@
     />
 
     <style>
-      .error-tag {
-        color: red;
-        font-style: italic;
-      }
+                body {
+                  margin: 0;
+                }
+
+                .container {
+                  margin: 8px;
+                }
+
+                .error-tag {
+                  color: red;
+                  font-style: italic;
+                }
+
+                .banner {
+                  width: 100vw;
+            text-align: center;
+            justify-content: center;
+      height: 48px;
+      line-height: 48px;
+                }
```

Comment:
suggestion: some of these attrs are redundant and we should specify line height as a unitless number[^1][^2]

```suggestion
                .banner {
                    text-align: center;
                    line-height: 3;
                }
```
OR
```suggestion
                .banner {
                    text-align: center;
                    line-height: 4;
                }
```

[^1]:
    > If the page is zoomed to increase the text size, using a unitless value ensures that the line height will scale proportionately.
    > 
    > -- https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#accessibility_concerns

[^2]: https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values

Reply:
RIGHT I remember this! Thank you!

Also I dunno what's up with the HEEx formatting - `mix format` uesd to work, but it seems like it's totally forgotten how HEEx code is supposed to look 😑

Reply:
[Here we go - this should fix both of our problems](https://github.com/mbta/skate/pull/2594/commits/6d2f2261a5f9f8c2f8163f03e69137360f2b9f1f)